Gruntfile.js

```diff
@@ -142,6 +144,8 @@ module.exports = function (grunt) {
 					"src/system-extension-contextual.js",
 					"src/system-extension-script-module.js",
 					"src/system-extension-steal.js",
+					// "src/system-extension-module-loaded-twice.js",
```

Comment:
Matthew and I added this line here as it appeared to have been accidentally left out. However, this caused all sorts of issues during bundling with steal tools (tested on project). I left it commented on purpose so that someone else in the future doesn't try to put it back.